Wheres-Waldo-Frontend/src/components/helpers/EndGame.jsx

```diff
@@ -0,0 +1,22 @@
+export default async function EndGame() {
+	const endGame = async () => {
+		const imageId = import.meta.env.VITE_IMAGE_ID;
```

Comment:
Shouldn't need this inner `endGame` function, you can just delete lines 2 and 21 and it should do the same thing.

And I'd change the name of the main function from `EndGame` to `endGame`, we reserve camel casing for components